Assets/GameAssets/Player/PlayerScripts/PlayerHealth.cs

```diff
@@ -21,7 +21,14 @@ public class PlayerHealth : BaseHealth
 
     //Variable is used by the dev console to determine whether the player should take damage or not
     public bool _shouldTakeDamage = true;//Nabil made this change
-
+    
+    [Tooltip ("Health point at which the heart beat sfx starts")]
+    [SerializeField] private float _heartToStartHeartSfx;
+    [Tooltip ("Health point at which the heart beat sfx ends")]
+    [SerializeField] private float _heartToEndHeartSfx;
+    [SerializeField] private float _heartBeatRateSfx;
+    private Coroutine _heartBeatCoroutine;
```

Comment:
you made this variable and aren't using it - make sure to assign this when the coroutine is started and use it to stop the coroutine